vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -7,8 +7,20 @@ L.OSM.TileLayer = L.TileLayer.extend({
   },
 
   initialize: function (options) {
+    isDarkMap = OSM.isDarkMap();
+    options.filter = (isDarkMap ? options.darkFilter : options.lightFilter) || options.filter || 'none';
     options = L.Util.setOptions(this, options);
-    L.TileLayer.prototype.initialize.call(this, options.url);
+    url = isDarkMap ? options.darkUrl : options.lightUrl;
```

Comment:
There is no `lightUrl` anywhere?

Reply:
Well, if the provided layers can be expected always to be a light-mode-first choice, the logic can be simplified to this more unreadable version:
```js
    options.filter = isDarkMap && options.darkFilter || 'none';
    options = L.Util.setOptions(this, options);
    url = isDarkMap && options.darkUrl;
    this.schemeClass = url && 'dark';
```
But I thought an implementation agnostic to that would be easier to think through with more options for cartographers' choices.